src/styles/App.style.ts

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import { device } from "./sizes";
+
+export const AppTitleNonFixed = styled.h1`
+text-align: center;
+`;
+
+export const AppBody = styled.div`
+padding-top: 75px;
+background: linear-gradient(-45deg, #e4b4a4, #e9bbcd, #77c0b0, #7db6aa);
+background-size: 400% 400%;
+-webkit-animation: gradient 15s ease infinite;
+animation: gradient 15s ease infinite;
+display: flex !important;
+flex-direction: column;
+justify-content: center !important;
+`
+
+export const AppContainer = styled.div`
```

Comment:
Also thought I remembered you asking me why I did this; was playing around with some styles, then assumed we'd probably eventually need to style it anyways so just left it in blank